src/js/messaging/containers/Folder.jsx

```diff
@@ -139,6 +141,7 @@ export class Folder extends React.Component {
 
   render() {
     const folderName = _.get(this.props.attributes, 'name');
+    const folderId = _.get(this.props.params, 'id', 0);
```

Comment:
The id in the URL corresponds to the folder being requested, and the one in attributes corresponds to the id of the actual current folder. This can get out of sync when the folder fails to load. It might be more intuitive to use the attributes id since that's the folder that's currently displayed.